### DISCUSSION TITLE
Regression / bug? (6.2)

### DISCUSSION THREAD

**Discussion by waran70** [YetiForceCompany/YetiForceCRM#14934] [General]
Cannot see button "Send email" in Campaigns module (Related lists).
Has been removed intentionally?

**Comment by rskrzypczak** [comment]
No, the button is still there.
Here you have the conditions that must be met for it to appear: https://github.com/YetiForceCompany/YetiForceCRM/blob/17fea2c3b1ce4f5756a9769151f2732d803ea079/modules/Campaigns/models/RelationListView.php#L27

**Reply by waran70** [reply]
Somehow I dont see how to activate button. It is not active in git too. Am I missing something?

**Reply by vovpff** [reply]
Let's check user rights (profile)

**Reply by rskrzypczak** [reply]
SMTP is not set on demo versions.
